### PR TITLE
Add [ONE1] Pool

### PR DESCRIPTION
**Pull request by joseiadicicco**
Please add my Pool [ONE1] to the Alliance.

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"ONE1",
+      "poolId":"pool1n2yp66zk6m33sg8k20hsvmk9tw7xeeswzdcfxhnx9hm9jkxv5k4",
```
Comment (lophre) [inline registry.json:4]: Your poolid is : 9a881d6856d6e31820f653ef066ec55bbc6ce60e1370935e662df659

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+      "social":{
+         "facebook":"",
+         "twitter":"@ADAOnePool",
+         "website":"https://adapool.one",
+         "reddit":"",
+         "instagram":"",
+         "youtube":"https://www.youtube.com/channel/UCaPUgQuIwYmQHea12Yen2ug"
+      },
+      "telegram":"https://t.me/joinchat/XVIgiKelvKFhOGQ5",
```
Comment (lophre) [inline registry.json:13]: You need to provide a Telegram usermame is this field, not a group.  Your telegram account will be used for SPA voting purposes.

**Review by lophre — CHANGES_REQUESTED** [review]
Please see comments

File: registry.json (hunk 2 above)
Comment (joseiadicicco) [reply registry.json:13]:
```suggestion
      "telegram":"@ADAONEPool",
```

File: registry.json (hunk 1 above)
Comment (joseiadicicco) [reply registry.json:4]:
```suggestion
      "poolId":"9a881d6856d6e31820f653ef066ec55bbc6ce60e1370935e662df659",
```

**Comment by lophre** [comment]
This one was pretty weird to merge.  I manually merge your change and you are now part of the Alliance!  Welcome in!

**Review by joseiadicicco — COMMENTED** [review]
I added the Telegram User (not group) and changed the Pool ID.

**Pull request closed without merging**